.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,7 +8,7 @@ https://github.com/blog/1506-closing-issues-via-pull-requests for more
 information.
 
 ### checklist
-- [ ] Make sure you read and understood the [CONTRIBUTING.md](https://github.com/xh3b4sd/anna/.github/CONTRIBUTING.md).
+- [ ] Make sure you read and understood the [CONTRIBUTING.md](/.github/CONTRIBUTING.md).
```

Comment:
I think this will not work from some pages, as it retrives the root of GitHub.com rather than the project root (i.e. `github.com/xh3b4sd/anna`). I may be mistaken however.